x-pack/plugins/canvas/public/components/workpad/workpad.js

```diff
@@ -63,84 +69,85 @@ export const Workpad = props => {
     }
   };
 
-  setDocTitle(workpad.name);
+  render() {
+    console.log('Workpad render');
+    const {
+      selectedPageNumber,
+      getAnimation,
+      onTransitionEnd,
+      pages,
+      totalElementCount,
+      width,
+      height,
+      workpadCss,
+      grid,
+      isFullscreen,
+    } = this.props;
 
-  return (
-    <div className="canvasWorkpad__buffer" style={bufferStyle}>
-      <div className="canvasCheckered" style={{ height, width }}>
-        {!isFullscreen && (
-          <Shortcuts name="EDITOR" handler={keyHandler} targetNodeSelector="body" global />
-        )}
+    const bufferStyle = {
+      height: isFullscreen ? height : height + 32,
```

Comment:
nit: can we make `32` a `const`?  It will help explain what it is and why it's necessary.

Reply:
I'm not sure why that value is there, I didn't change that. I'll try figuring it out though.